addons/sourcemod/scripting/MyJailbreak/ratio.sp

```diff
@@ -90,7 +90,7 @@ Handle hKeyValues;
 // Integer
 int g_iRandomAnswer[MAXPLAYERS+1];
 int g_iQuestionTimes[MAXPLAYERS+1];
-int countQuestions = 0;
+int countQuestions;
```

Comment:
please name it **g_i**CountQuestions or **g_i**QuestionsCount

'g_' stand for global variable
'i' stand for a integer varible